_data/simple-icons.json

```
@@ -9034,6 +9034,11 @@
             "hex": "792EE5",
             "source": "https://github.com/Lightning-AI/lightning/blob/a584196abf820179adb0758ef67ddae91c44e7bc/docs/source/_static/images/icon.svg"
         },
+        {
+            "title": "Lime",
+            "hex": "00dd00",
```

Hex codes must be capitalized. Please run `npm run lint` locally to catch mistakes like this before raising a PR.
```suggestion
            "hex": "00DD00",
```